acra/security-controls/security-logging-and-events/_index.md

```diff
@@ -20,6 +45,12 @@ bookCollapseSection: true
 
 ## Security events
 
+* Poison record access
+
+  Access to data inside database that was not meant to be accessed at all.
+  These poison records can be inserted to catch situations like malicious SQL client attempting to dump the whole database.
```

Comment:
not only SQL clients. AcraTranslator supports poison records too. So poison records can be catched on data not only from database, but from any storage where it is saved in case with AT>